tla-bmcmt/src/main/scala/at/forsyte/apalache/tla/bmcmt/rewriter/ConstSimplifierForSmt.scala

```diff
@@ -5,14 +5,17 @@ import at.forsyte.apalache.tla.bmcmt.smt.SolverContext
 import at.forsyte.apalache.tla.lir._
 import at.forsyte.apalache.tla.lir.oper._
 import at.forsyte.apalache.tla.lir.values.{TlaBool, TlaInt}
+import at.forsyte.apalache.tla.pp.ConstSimplifierBase
 
 /**
-  * A simplifier of constant TLA+ expressions, e.g., rewriting 1 + 2 to 3.
-  * This simplifier is using some knowledge about SMT.
+  * <p>A simplifier of constant TLA+ expressions, e.g., rewriting 1 + 2 to 3.
+  * This simplifier is using some knowledge about SMT.</p>
+  *
+  * <p>Bugfix #450: make sure that the integers are simplified with BigInt.</p>
```

Comment:
I'm not sure I see the value of putting bugfix history in the documentation. Is this common practice in Java/Scala land?

Reply:
Well, I don't put every bugfix in the code. Sometimes, I add them, not to repeat the same mistake twice :-)